integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeApi.kt

```diff
@@ -125,7 +128,6 @@ class CrowdNodeApiAggregator @Inject constructor(
     override var showNotificationOnResult = false
 
     init {
-        restoreStatus()
```

Comment:
Not that we call this method from `StakingActivity` anyway, it can be made `suspend` to account for fetching `GetFunds` and removed from the ctor.